addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -228,11 +236,26 @@ options.registry.blog_cover = options.Class.extend({
 
         _.each(this.$el, function (el) {
             var $el = $(el);
-            $el.toggleClass('d-none',
-                $el.is(':not([data-change])') && !self.$target.hasClass('cover')
-                || $el.is(':has([data-select-class])') && self.$target.hasClass('o_list_cover'));
+
+            if ($el.is(':not([data-change])')) {
+                $el.removeClass('d-none');
+
+                _.each(['size','filters','text_size', 'text_align'], function (opt_name) {
+                    var $opts = $el.find('[data-coverOpt="'+ opt_name +'"]');
+                    var notAllowed = (self.$target.data('use_' + opt_name) != 'True');
+
+                    if ($opts.length && (!self.$target.hasClass('o_wblog_has_cover') ||  notAllowed) ) {
+                        $el.addClass('d-none');
+                    }
+                });
+            }
+
+            $el.find('.o_js_hide_regular').toggleClass('d-none', self.$target.is('.o_wblog_post_page_cover_regular'));
+            $el.find('.o_js_show_regular').toggleClass('d-none', !self.$target.is('.o_wblog_post_page_cover_regular'));
```

Comment:
Condition is duplicated (CTRL-X! :p)

Reply:
Hide A if condition
Hide B if **not** condition

I may replace it with
```
Y = condition
A toggleClass (hide, if Y)
B toggleClass (hide, if not Y)
```

it's what you suggest?

Reply:
Yes ;)